extensions/big-data-cluster/src/bigDataCluster/dialog/mountHdfsDialog.ts

```diff
@@ -390,15 +390,24 @@ export class MountHdfsDialog extends HdfsDialogBase<MountHdfsProperties> {
 				{
 					component: this.pathInputBox,
 					title: hdfsPathTitle,
-					required: true
+					required: true,
+					layout: {
+						info: localize('mount.hdfsPath.info', "Path to a new (non-existing) directory which you want to associate with the mount")
+					}
 				}, {
 					component: this.remoteUriInputBox,
-					title: localize('mount.remoteUri', "Remote URI"),
-					required: true
+					title: localize('mount.remoteUri.title', "Remote URI"),
+					required: true,
+					layout: {
+						info: localize('mount.remoteUri.info', "The URI to the remote data source. Example for ADLS: abfs://fs@saccount.dfs.core.windows.net/")
```

Comment:
Should we point to a docs page instead? Is ADLS included because it's going to be the typical use case here? Also, any reason for "saccount" instead of "account"?

Reply:
Yes, ADLS is a common option. You can remove the specific example if you want. Let's just do that.

Reply:
I actually like it, is it OK to leave for now? It's helpful to know what to look for.